addons/website_event/views/event_templates_list.xml

```diff
@@ -203,7 +203,7 @@
                             </time>
                             <!-- Location -->
                             <div itemprop="location" t-field="event.address_id" t-options="{'widget': 'contact', 'fields': ['city'], 'no_marker': 'true'}"/>
-                            <div class="mt8 d-flex align-items-center">
+                            <div class="mt8 d-table align-items-center">
```

Comment:
Counter-proposal:

Remove the "d-table" and also the "align-items-center" (that doesn't do anything anymore).
Change "mr8" to "mr4" on tags.

Seems to give the same display without using a strange d-table (to not display a real table).

Can you give it a try?